### ISSUE TITLE
Add missing areas

### ISSUE THREAD

**Issue by juangallostra** [MadBoulder/BetaLibrary#79]
At the time of writing:
- [ ] Arenys de Munt
- [ ] Céllecs
- [ ] Dorres/Les Escaldes
- [ ] Enveitg
- [ ] La Cabrera
- [ ] Laramade
- [ ] Lynn Valley Boulders
- [ ] Mens
- [ ] Peña Corneira
- [ ] SFU Boulder
- [ ] Salvan
- [ ] Sant Elm
- [ ] Tavertet
- [ ] Val Bavona
- [ ] Valdemanco
- [ ] Vilajuïga
- [ ] Xistral
- [ ] Xixona

**Comment by asantacreu** [comment]
Some areas are not added on purpose...

**Comment by juangallostra** [comment]
@asantacreu the areas that are not included cannot be accessed through the web. Maybe we can think a way of making them accessible? Or it isn't worth it?

**Comment by asantacreu** [comment]
I don't think it is woth it.. I prefer to wait until we have more relevant content of the area, and then it makes sense to add them.
Little zones with just one video aren't relevant enough to be counted as area supported.

That said, there are some areas int the list that could be added:
- [ ] Lynn Valley Boulders
- [ ]  SFU Boulder
- [ ]  Salvan

**Comment by asantacreu** [comment]
In any case, it is good to have a list of the areas that are not on the website... maybe it can be added on some way to the statistics?

**Comment by juangallostra** [comment]
Ok! I'll keep the list here anyway so that we can keep track of which areas haven't been added yet.

**Comment by juangallostra** [comment]
@asantacreu maybe we can establish some criteria to determine when a zone should be included? By setting a number of videos or something like that. This way I can automatically add a new zone when I see that the condition is met. 

What do you think?

**Comment by asantacreu** [comment]
@juangallostra Yeah, we should have some criteria. Because I don't want Arenys de Munt added because it is not relevant enough, I'd say the minumum is 6 videos or higher.

For the moment we can add Cimaganda, Val Bavona and Xixona.

**Comment by juangallostra** [comment]
@asantacreu Cimaganda, Val Bavona and Xixona added!

**Comment by juangallostra** [comment]
Add Zarzalejo and Chaos Canyon
